indico/modules/events/util.py

```diff
@@ -415,11 +415,11 @@ def get_base_ical_parameters(user, detail, path, params=None):
             'user_logged': user is not None, 'request_urls': request_urls}
 
 
-def create_event_logo_tmp_file(event):
+def create_event_logo_tmp_file(event, dir=config.TEMP_DIR):
```

Comment:
call it `path` to avoid shadowing the builtin `dir`

Reply:
Done